cmake/modules/curl.cmake

```diff
@@ -77,6 +77,9 @@ else()
 			BUILD_IN_SOURCE 1
 			BUILD_BYPRODUCTS ${CURL_LIBRARIES}
 			INSTALL_COMMAND "")
+		install(FILES "${CURL_LIBRARIES}" DESTINATION "${CMAKE_INSTALL_LIBDIR}/${LIBS_PACKAGE_NAME}/")
+		install(DIRECTORY "${CURL_INCLUDE_DIR}/curl" DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${LIBS_PACKAGE_NAME}/"
```

Comment:
I am just curious: in other cmake modules you added the trailing "/" to the `INCLUDE` variables; here instead it ends without the "/": `${CURL_INCLUDE_DIR}/curl`.
Shouldn't this be: `${CURL_INCLUDE_DIR}/curl/`?